pytest_fixtures/core/contenthosts.py

```diff
@@ -6,6 +6,7 @@
 """
 import pytest
 from broker import VMBroker
+from nailgun import entities
```

Comment:
this can be removed now that it is no longer used

Reply:
OK